test/evergreen/code_complexity_analysis.py

```diff
@@ -0,0 +1,118 @@
+# importing the module
+import argparse
+import json
+import csv
+import os.path
+import ast
+from heapq import nlargest
+import pandas as pd 
+from collections import defaultdict
+    
+class CodeComplexity:
+    dataFrame = pd.DataFrame()
+    topNRegions = 5
+    def __init__(self):
+        self.parser = argparse.ArgumentParser()
+        self.parser.add_argument('-s', '--summary', required=True, help='Path to the complexity summary data file')
+        self.parser.add_argument('-o', '--outfile', help='Path of the file to write test output to')
+        self.parser.add_argument('-d', '--data_file', help='Code complexity data file')
+
+    def run(self):
+        self.args = self.parser.parse_args()
+        self.dataFrame = pd.read_csv(self.args.data_file)
+
+        self.get_atlas_compatible_code_statistics(self.args.summary, self.args.data_file, self.args.outfile)
+
+    # Generate the Atlas compatible format report.
+    def get_atlas_compatible_code_statistics(self, summaryFile, dataFile, outfile='atlas_code_statistics.json'):
+        atlas_format = {
+                    'Test Name': "Code Statistics",
```

Comment:
Should the test name be "Code Complexity" to be distinguished from the existing "Code Coverage" test name?

Reply:
Fixed in 7cc186a02197e388c9a2fcaa97331d24c84376a5